mesonbuild/build.py

```diff
@@ -2107,6 +2117,9 @@ def should_install(self):
     def get_filename(self):
         return self.name
 
+    def get_outputs(self):
```

Comment:
Actually, the concept of outputs does not make much sense for a run target.
But you have been right, that self.name should only be a string here.

We should revisit this in a later PR, because having a output file name for a run target doesn't even make sense (if there is one, meson does not know about it)

Reply:
Maybe return an empty list in the future, when we change the `filename` key to a list?